build.gradle.kts

```
@@ -14,9 +14,12 @@ dependencies {
 
     implementation(platform("org.openrewrite:rewrite-bom:${rewriteVersion}"))
     implementation("org.openrewrite:rewrite-java")
-    implementation("org.openrewrite:rewrite-groovy:${rewriteVersion}")
-    implementation("org.openrewrite:rewrite-kotlin:${rewriteVersion}")
-    implementation("org.openrewrite:rewrite-csharp:${rewriteVersion}")
+    compileOnly("org.openrewrite:rewrite-groovy:${rewriteVersion}")
+    testImplementation("org.openrewrite:rewrite-groovy:${rewriteVersion}")
```

This was already in the original, but the version of rewrite-groovy is managed by rewrite-bom.
```suggestion
    compileOnly("org.openrewrite:rewrite-groovy")
    testImplementation("org.openrewrite:rewrite-groovy")
```